pkg/scheduler/framework/types.go

```diff
@@ -75,15 +75,75 @@ const (
 	WildCard              GVK = "*"
 )
 
-// WildCardEvent semantically matches all resources on all actions.
-var WildCardEvent = ClusterEvent{Resource: WildCard, ActionType: All}
+var (
```

Comment:
I think we should put those in `pkg/scheduler/internal/queue/events.go` since they are not used by the plugins.